Gruntfile.js

```diff
@@ -212,6 +212,35 @@ module.exports = function(grunt) {
           dest: 'tmp',
           ext: '.css'
         }]
+      },
```

Comment:
Added more configurations for running stylus. One producing an external sourcemap file and another one inline.